lib/ansible/module_utils/common/validation.py

```diff
@@ -39,6 +40,10 @@ def count_terms(terms, parameters):
 
 
 def safe_eval(value, locals=None, include_exceptions=False):
+    deprecate(
+        "ansible.module_utils.common.safe_eval function is no longer used.",
```

Comment:
Since this message will be used for both direct callers and indirect callers, it's probably better to be less specific:

```suggestion
        "The safe_eval function should not be used.",
```